### DISCUSSION TITLE
Ogmios 6 client npm package

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#357] [Q & A] [answered]
Hello.
What is the plan for publishing the [@cardano-ogmios/client 6.0.0](https://www.npmjs.com/package/@cardano-ogmios/client?activeTab=versions) npm package.
I could find the version, but it seems to be very old, while the ogmios 6 stable version has only just been released.
Thank you!

**Comment by KtorZ** [comment, marked as answer]
Arg. That existing 6.0.0 package should be versioned 6.0.0-rc1; so the push of the last one failed. I'll replace the versions on npm; but this will take ~24h.

**Reply by mirceahasegan** [reply]
I suppose that [breaking changes in 6.0](https://github.com/CardanoSolutions/ogmios/releases/tag/v6.0.0), like `All values are now returned as a map of maps`, are not visible/usable until the client is updated, right?

**Reply by KtorZ** [reply]
Correct.

**Reply by mirceahasegan** [reply]
thank you for the quick answers 🥇
